explainer.md

```diff
@@ -18,96 +18,107 @@ permanent "block" policies, or
-   the unmute buttons. Because the user agent has no insight into this interaction
-   in the content area, it is compelled to respect the user's previous decision.
-   Especially in a stressful scenario such as an important presentation, users will
-   struggle to navigate the settings surfaces to change the permission decision._
-
-Overall, attempts to optimize the trade-off between usability and interruptions
-have hit practical limits because, fundamentally, user agents
+There are four main challenges with the status quo:
+
+1.  **Insufficiency of existing mitigations** The present day permissions spam
+    and abuse mitigation approach has an architectural upper bound on user
+    protection because the model relies on the website to choose when to trigger
+    the permission request prompt rather than capturing a reliable signal of
+    user intent. Requiring a user gesture for the Permission API (or similar)
+    does not solve this problem as there are many ways of tricking a user into
+    providing such a gesture.
+
+1.  **Context**: Ideally, a site's developer will request access as part of a
+    contextual flow that helps users understand what's being asked for and why,
+    enabling quick and confident responses. Often, however, permission requests
+    are correlated poorly with user expectations, up to and including prompts
+    that can come out of nowhere (see example 1). This places a burden on user
+    agents' presentation of the request. The user agent has no semantic
+    understanding of events taking place in the content area prior to the
+    permission request. User agents could make better decisions and provide
+    better prompts if they could make well-founded assumptions about the nature
+    of the user's interaction in the content area, and the user's intent.
+
+    ![](images/image1.png) \
+    *Example 1. A notification permission prompt on a news site (contents
+    blurred), shown after the user has clicked on the empty area next to the
+    article content. The user finds this prompt interruptive as they had no
+    interest in subscribing to notifications, and they will likely struggle to
+    understand why the prompt was shown to begin with.*
+
+1.  **Location**: In the ideal case above, users will interact with something on
+    a site that triggers a prompt. In less ideal cases, the user might not have
+    interacted with anything at all, or they may have interacted with an element
+    that was unrelated to the request. Given this uncertainty, user agents rely
+    on common placement of the permission prompt, usually in the top-left of the
+    page. Even in the best case, this has the unfortunate effect of shifting the
+    point to which users need to pay attention from the thing they clicked on to
+    some distant part of the user agent's UI (see example 2). User agents could
+    make better decisions and provide better prompts if they could make
```

Comment:
General feedback... There is definitely scope to do something different, specially on desktop browsers... but it's feels a bit presumptuous to assume the general UX model will change. Increasingly (on Apple's platforms at least), the permissions requests experience (UX and UI) are all governed by the underlying OS. There are some exceptions, like with getUserMedia(), but even there, the permission to access the camera and microphone are still OS level prompts I believe.

Reply:
Extracted into https://github.com/WICG/PEPC/issues/8